editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -111,7 +123,7 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 		try {
 			prj.refreshLocal(IProject.DEPTH_INFINITE, subMonitor.newChild(1));
 			// update the manifest to reflect project name as Bundle-SymbolicName
-			updateBundleSymbolicName(prj, subMonitor.newChild(1));
+			updateBundlePluginConfiguration(prj, subMonitor.newChild(1));

```

Comment:
Ithink that if pom is well-configured and m2e connector improved this would be possible